.yarnrc

```diff
@@ -3,9 +3,3 @@ yarn-offline-mirror ".yarn-local-mirror"
 
 # Always look into the cache first before fetching online
 --install.prefer-offline true
-
-# Disable interactive and progress logs as yarn install is now
-# managed by Bazel and we are piping the logs from the underlying
-# process running bazel into the parent one running kbn
---install.non-interactive true
```

Comment:
I think we should recover those options. Without them the Playwright browsers installation is too noisy

Reply:
The install log doesn't come from here. It comes from the fact that we're now reflecting the logging asks of the bootstrap command inside the subcommands. It was completely muted before. I can mute the playwright install